demisto_sdk/commands/common/errors.py

```diff
@@ -4345,3 +4371,43 @@ def nativeimage_exist_in_script_yml(script_id):
             f"script {script_id} contains the nativeimage key in its yml, "
             f"this key is added only during the upload flow, please remove it."
         )
+
+    @staticmethod
+    @error_code_decorator
+    def uses_items_not_in_marketplaces(
+        content_name: str, marketplaces: list, content_items: list
+    ):
+        return (
+            f"Content item '{content_name}' with marketplaces: '{', '.join(marketplaces)}' is using content items: "
+            f"'{', '.join(content_items)}' that have incompatible marketplaces."
```

Comment:
content item .. can be used in the .. marketplaces, however it uses ... which is not supported in the ... marketplace